man/unattended-upgrade.8

```diff
@@ -38,8 +39,18 @@ help output
 \fB-d\fR, \fB\-\-debug\fR
 extra debug output into /var/log/unattended-upgrades.log
 .TP
+\fB--apt-debug\fR
+detailled APT/LibAPT output for debugging
+.TP
+\fB-v\fR, \fB\-\-verbose\fR
+show verbose output
+.TP
 \fB--dry-run\fR
 Just simulate installing updates, do not actually do it
+.TP
+\fB--minimal-upgrade-steps\fR
+perform upgrade in minimal steps (cancel with SIGINT)
```

Comment:
Also please add --no-minimal-upgrade-steps description, since minimal steps is the default no-minimal-upgrade-steps became more important.